api/metrics/community.py

```diff
@@ -149,3 +149,20 @@ def covid_spread_sum_by_area(self, column_name):
         cur.execute(query)
         rows = rows_to_dicts(cur, cur.fetchall())
         return rows
+
+    def belonging(self, year, segment):
+        """
+        Returns a value between 0 and 1, the rate of residents who feel a sense of belonging for each community area
+        """
+        query = """
+        SELECT
+            area_number,
+            round((CAST(value AS REAL))/100, 3) AS value
```

Comment:
We don't have to round the result on the backend, we can do that on the frontend.

To make your test work, instead of writing out the float, you could write the actual division:

```python
{ "area_number": 1, "value": 45.6 / 100 },
```

Let me know if that works!

Reply:
This worked perfectly for the unit test. I also removed the casting to a REAL since table schema already has it defined as such.